src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -243,4 +258,85 @@ export class MemberListComponent implements OnChanges, OnDestroy {
         }
       });
   }
+
+  onRemoveGroup(event: Event, selectedIds: string[]): void {
+    this.confirmationService.confirm({
+      target: event.target || undefined,
+      key: 'commonPopup',
+      icon: 'pi pi-exclamation-triangle',
+      message: $localize`Are you sure you want to permanently delete ${getSelectedGroupChildCaptions(this.selection as GroupChild[])}?
+       This operation cannot be undone.`,
+      acceptLabel: $localize`Yes`,
+      acceptIcon: 'fa fa-check',
+      rejectLabel: $localize`No`,
+      accept: () => this.removeGroupsOrSubgroups(selectedIds),
+    });
+  }
+
+  onRemoveSubgroups(event: Event, groupId: string, selectedIds: string[]): void {
+    this.confirmationService.confirm({
+      target: event.target || undefined,
+      key: 'commonPopup',
+      icon: 'pi pi-exclamation-triangle',
+      message: $localize`By removing ${getSelectedGroupChildCaptions(this.selection as GroupChild[])} from the group, you may loose
+       manager access to them (if no explicit permission or through other parent group). Are you sure you want to proceed?`,
+      acceptLabel: $localize`Yes`,
+      acceptIcon: 'fa fa-check',
+      rejectLabel: $localize`No`,
+      accept: () => this.removeGroupsOrSubgroups(selectedIds, groupId),
+    });
+  }
```

Comment:
To me `selectedIds` induces confusion, you have it directly from the class property `selection`, there's no need to make it a method argument
Or I am missing something :monocle_face:

Reply:
Fixed